src/profile-logic/call-tree.js

```diff
@@ -209,6 +214,41 @@ export class CallTree {
     };
   }
 
+  _getInliningBadge(
```

Comment:
I like this new badge a lot! Thanks for adding it! As you wrote in the commit message, it's a great answer to "Why wasn't this inlined!?" 😄